### PR TITLE
Rename variable name to uppercase NAME

### PR DESCRIPTION
**Pull request by radoondas**
This is just a minor naming update of the script.
Is there any reason for variable 'name' to be lowercase?

### CODE REVIEW

**Comment by damm** [comment]
is there any reason to have name in uppercase either? assignment of variables is case sensitive in shells as well (typically).

File: load.sh [hunk 1]
```diff
@@ -87,37 +87,37 @@ echo "Loading dashboards to $ELASTICSEARCH in $KIBANA_INDEX"
 
 for file in $DIR/search/*.json
 do
-    name=`basename $file .json`
-    echo "Loading search $name:"
-    $CURL -XPUT $ELASTICSEARCH/$KIBANA_INDEX/search/$name \
+    NAME=`basename $file .json`
+    echo "Loading search $NAME:"
+    $CURL -XPUT $ELASTICSEARCH/$KIBANA_INDEX/search/$NAME \
```
Comment (damm) [inline load.sh]: We could be very careful here and make it ${NAME}

File: load.sh [hunk 2]
```diff
@@ -87,37 +87,37 @@ echo "Loading dashboards to $ELASTICSEARCH in $KIBANA_INDEX"
 
 for file in $DIR/search/*.json
 do
-    name=`basename $file .json`
-    echo "Loading search $name:"
-    $CURL -XPUT $ELASTICSEARCH/$KIBANA_INDEX/search/$name \
+    NAME=`basename $file .json`
+    echo "Loading search $NAME:"
```
Comment (damm) [inline load.sh]: ${NAME} as well

**Comment by radoondas** [comment]
Well yes. On the other hand I would suggest not only ${NAME}, but more curly braces where variables are in the strings to be more safe and consistenst within the script. I will create PR later, with additions.

File: load.sh (hunk 2 above)
Comment (ruflin) [inline load.sh]: +1 on using ${NAME} in general. Just do be save.

**Comment by ruflin** [comment]
@radoondas Thanks

**Comment by radoondas** [comment]
For the reference, I have prepared another commit to  my fork with curly braces in the script: 6e1948d

I have no PR so far. I believe, further discussion is required.

**Comment by radoondas** [comment]
I believe, this one can be just closed. It is obsolete since #86 is now merged and #85 is in discussion.

**Comment by ruflin** [comment]
Agree in closing this as changes are in #86 and #85

**Pull request closed without merging**